src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -119,7 +112,11 @@ const ColonyHome = ({
     COLONY_TOTAL_BALANCE_DOMAIN_ID,
   );
   const [isTaskBeingCreated, setIsTaskBeingCreated] = useState(false);
-  const [showRecoverOption, setRecoverOption] = useState(false);
+  /*
+   * @TODO Re-add domains once we decide what we're going to do with the
```

Comment:
As stated in the other comment, let's just delete this and everything it's using (saga, command, etc)